shadow-robot/sr-ros-interface
# Add the missing mesh for the finger biotac adapter.

Not tested.

To test (on indigo):

export BIOTAC_HAND=1
roslaunch sr_hand gazebo_hand.launch

And check that the biotac adapters (not the same as the biotac sensors) show in the gazebo simulation.
